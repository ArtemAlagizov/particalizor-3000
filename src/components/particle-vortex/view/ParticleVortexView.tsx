import React from "react";
import { Styled } from "../styled";
import { IParticleVortexViewProps } from "./IParticleVortexViewProps";

const ParticleVortexView: React.FC<IParticleVortexViewProps> = ({
  destinationCanvasRef
}) => {
  return (
    <Styled.DestinationCanvas>
      <canvas
        ref={destinationCanvasRef}
        className="particalizor-3000-destination-canvas"
      />
    </Styled.DestinationCanvas>
  );
};

export default ParticleVortexView;
