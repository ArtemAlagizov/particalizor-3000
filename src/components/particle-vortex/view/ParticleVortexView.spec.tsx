import React from "react";
import { shallow } from "enzyme";
import ParticleVortexView from "./ParticleVortexView";

describe("Given ParticleVortexView", () => {
  describe("when rendering", () => {
    it("should have ParticleVortexView matching snapshot", () => {
      const component = shallow(
        <ParticleVortexView destinationCanvasRef={null} />
      );

      expect(component).toMatchSnapshot();
    });
  });
});
