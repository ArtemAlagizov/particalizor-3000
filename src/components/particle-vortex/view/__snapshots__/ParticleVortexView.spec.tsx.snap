// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given ParticleVortexView when rendering should have ParticleVortexView matching snapshot 1`] = `
<styled.div>
  <canvas
    className="particalizor-3000-destination-canvas"
  />
</styled.div>
`;
